website/src/components/Changelog/Gateway.tsx

```diff
@@ -8,6 +8,14 @@ export default function Gateway() {
 
   return (
     <Entries href={href} arches={arches} title="Gateway">
+      <Entry version="1.1.X" date={new Date("2024-XX-XX")}>
```

Comment:
This was supposed to be commented-out, oopsy https://firezonehq.slack.com/archives/C04HRQTFY0Z/p1723131714460069